js/app.js

```diff
@@ -1,2 +1,44 @@
 $(function () {
+  $("#search").submit(() =>{
+    event.preventDefault()
+
+
+    console.log("submitting form")
+    // read value from text field
+    //store in query variable
+    const query = $("#query").val()
+    console.log(query)
+    search(query)
+
+  })
+
+  function search(term){
+    //this is where we make our api request
+    console.log(`calling search with q=${term}`)
+    const url = "https://api.openweathermap.org/data/2.5/weather"
+    const apiKey = "b1fc5e5396249435c53ad57e7f01f7bf"
+
+    jQuery.ajax({
+      url: url,
+      type: "GET",
+      data: { q: term, api_key: apiKey}
```

Comment:
@cbrooks2019 Looks like you're using the incorrect property here to represent the API key. You're using `api_key`  however, OpenWeatherMap's API documentation calls for developers to use `app_id` as the property needed to represent the API key for their service. 

`data: { q: term, appid: apiKey}` instead of what you currently have: `data: { q: term, api_key: apiKey}`

Also, this property is required by the API, so not sending will always result in an error. 

Unfortunately, OpenWeatherMap's API documentation doesn't make it easy to determine this but with a little digging, they do provide an example. See the docs here: https://openweathermap.org/appid

As a developer, we have to deal with good and bad documentation. In this case, the docs weren't great. 



See to

Reply:
@cbrooks2019 I clone your repo and ran your code after making the change described above (using `appid` instead of `api_key`) and I was able to successfully get data back!